### PR TITLE
chore: increase max proof window

### PR DESCRIPTION
**Pull request by onbjerg**
This increase is to accomodate 1 week of data on 2 second block times on Optimism

### CODE REVIEW

**Review by shekhirin — APPROVED** [review]

File: crates/rpc/rpc-server-types/src/constants.rs [hunk 1]
```diff
@@ -48,8 +48,9 @@ pub const DEFAULT_ENGINE_API_IPC_ENDPOINT: &str = "/tmp/reth_engine_api.ipc";
 /// The default eth historical proof window.
 pub const DEFAULT_ETH_PROOF_WINDOW: u64 = 0;
 
-/// Maximum eth historical proof window. Equivalent to roughly one month of data.
-pub const MAX_ETH_PROOF_WINDOW: u64 = 216_000;
+/// Maximum eth historical proof window. Equivalent to roughly one and a half months of data on a 12
+/// second block time, and a week on a 2 second block time.
+pub const MAX_ETH_PROOF_WINDOW: u64 = 302_400;
```
Comment (mattsse) [inline crates/rpc/rpc-server-types/src/constants.rs:53]: could we use math here? 
like seconds month / blocktime?

**Review by mattsse — APPROVED** [review]
lgtm, suggesting maths

File: crates/rpc/rpc-server-types/src/constants.rs (hunk 1 above)
Comment (onbjerg) [reply crates/rpc/rpc-server-types/src/constants.rs:53]: we could, although what blocktime do we base it off?

**Review by rkrasiuk — APPROVED** [review]

File: crates/rpc/rpc-server-types/src/constants.rs (hunk 1 above)
Comment (rkrasiuk) [reply crates/rpc/rpc-server-types/src/constants.rs:53]: originally we based it off `30 * 24 * 60 *60 / 12`. now it's `7 * 24 * 60 *60 / 2`

**Pull request merged**